pkg/config/tls.go

```diff
@@ -18,16 +18,21 @@ type TLSConfig struct {
 	CertFile string `json:"cert_file" toml:"cert_file" yaml:"cert_file"`
 }
 
-func InitTLS(sslMode, certFile, keyFile string) (*tls.Config, error) {
-	if sslMode != SSLMODEDISABLE {
-		tracelog.InfoLogger.Printf("loading tls cert file %s, key file %s", certFile, keyFile)
-		cert, err := tls.LoadX509KeyPair(certFile, keyFile)
-		if err != nil {
-			return nil, xerrors.Errorf("failed to load tls conf: %w", err)
-		}
-		return &tls.Config{Certificates: []tls.Certificate{cert}, InsecureSkipVerify: true}, nil
-	} else {
+func (c *TLSConfig) IsTLSModeDisable() bool {
+	return c == nil || c.SslMode == SSLMODEDISABLE || (c.CertFile == "" && c.KeyFile == "" || c.SslMode == "")
+}
+
+func (c *TLSConfig) Init() (*tls.Config, error) {
+	tracelog.InfoLogger.Printf("Init TLS kek")
+	if c.IsTLSModeDisable() {
 		tracelog.InfoLogger.Printf("skip loading tls certs")
+		return nil, nil
+	}
+
+	tracelog.InfoLogger.Printf("loading tls cert file %s, key file %s", c.CertFile, c.KeyFile)
```

Comment:
sqprlog here and above

Reply:
well, using spqrlog here causes import cycle because spqrlog requires config.
I suggest getting log level not from config but from environment variable or pass log level to logger and every service should create its own logger with its own level from its own config. 
But maybe fix this next time ¯\_(ツ)_/¯